### ISSUE TITLE
NPE 

### ISSUE THREAD

**Issue by Shadowrs** [glisco03/deathlog#11]
AOF5 modpack 1.2.3 deathlog 0.2.9+1.18 maybe its already fixed in newer version

```
java.lang.NullPointerException: Cannot read field "screenshotsEnabled" because the return value of "com.glisco.deathlog.client.Config.instance()" is null
	at com.glisco.deathlog.client.gui.DeathLogScreen.createScreenshotButtonText(DeathLogScreen.java:72)
	at com.glisco.deathlog.client.gui.DeathLogScreen.method_25426(DeathLogScreen.java:62)
	at net.minecraft.class_437.redirect$fij000$onInit(class_437.java:8790)
	at net.minecraft.class_437.method_25423(class_437.java:389)
	at net.minecraft.class_310.method_1507(class_310.java:993)
	at com.glisco.deathlog.client.DeathLogClient.openScreen(DeathLogClient.java:57)
	at com.glisco.deathlog.client.DeathLogClient.openScreen(DeathLogClient.java:52)
	at com.glisco.deathlog.client.DeathLogClient.lambda$onInitializeClient$0(DeathLogClient.java:36)
	at net.minecraft.class_4185.method_25306(class_4185.java:40)
	at net.minecraft.class_4264.method_25348(class_4264.java:16)
	at net.minecraft.class_339.method_25402(class_339.java:121)
	at net.minecraft.class_4069.method_25402(class_4069.java:27)
	at net.minecraft.class_312.method_1611(class_312.java:94)
	at net.minecraft.class_437.method_25412(class_437.java:492)
	at net.minecraft.class_312.method_1601(class_312.java:94)
	at net.minecraft.class_312.method_22686(class_312.java:165)
	at net.minecraft.class_1255.execute(class_1255.java:103)
	at net.minecraft.class_312.method_22684(class_312.java:165)
	at org.lwjgl.glfw.GLFWMouseButtonCallbackI.callback(GLFWMouseButtonCallbackI.java:36)
	at org.lwjgl.system.JNI.invokeV(Native Method)
	at org.lwjgl.glfw.GLFW.glfwPollEvents(GLFW.java:3101)
	at com.mojang.blaze3d.systems.RenderSystem.flipFrame(RenderSystem.java:197)
	at net.minecraft.class_1041.method_15998(class_1041.java:310)
	at net.minecraft.class_310.method_1523(class_310.java:1147)
	at net.minecraft.class_310.method_1514(class_310.java:737)
	at net.minecraft.client.main.Main.main(Main.java:236)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:460)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
```

**Comment by gliscowo** [comment]
Deleting the config should fix this problem, although it is quite concerning that it still happens. I'll look into it

**Comment by Shadowrs** [comment]
log was 0kb, yep restarting the pack re-generated it

**Comment by gliscowo** [comment]
In case you wanna get the update, version [0.2.10](https://modrinth.com/mod/deathlog/version/A3jmeepA) fixes this problem (although it is only available for 1.19)
Cheers, and thanks for reporting
